llama.cpp

```diff
@@ -1757,6 +1758,12 @@ llama_token llama_sample_top_p_top_k(
     // TODO: avoid this ...
     const auto last_n_tokens = std::vector<llama_token>(last_n_tokens_data, last_n_tokens_data + last_n_tokens_size);
 
+    if (std::abs(temp) < FLT_EPSILON) {
+        temp = 0.8f;
+        top_k = 1.0f;
```

Comment:
I believe this should be `top_k = 1;` as it's an integer.

Reply:
You are correct. Thanks.